README.md

```diff
@@ -160,7 +160,7 @@ See the section about [running tests](https://facebook.github.io/create-react-ap
 
 Contributions, issues, and feature requests are welcome!
 
-Feel free to check the [issues page](../../issues/).
+Feel free to check the [issues page](https://github.com/mohamedSabry0/math-geeks/issues).
```

Comment:
- [optional] kindly remove the live demo link since your application has not yet been deployed
- [optional] please kindly add the section Future Features, including 1-3 potential features to be added so as to improve your work